pydantic/errors.py

```diff
@@ -324,6 +324,14 @@ def __init__(self, *, expected_arbitrary_type: AnyType) -> None:
         super().__init__(expected_arbitrary_type=display_as_type(expected_arbitrary_type))
 
 
+class ClassError(PydanticTypeError):
+    code = 'class'
```

Comment:
Probably better to define `SubclassError` like this and then `ClassError` for bare `Type` which can have a simple message with no variables.